### PR TITLE
Addition of Morrisons

### PR DESCRIPTION
**Pull request by djrarky**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -374,6 +374,7 @@ websites:
       img: natwest.png
       tfa: Yes
       hardware: Yes
+      software: Yes
       doc: http://www.natwest.com/personal/online-banking/g1/banking-safely-online/card-reader.ashx
```
Comment (stephengroat) [inline _data/banking.yml:377]: we try to do limit site addition PRs to not edit existing sites, please remove changes

File: _data/banking.yml [hunk 2]
```diff
@@ -666,4 +667,3 @@ websites:
       software: Yes
       hardware: Yes
       doc: https://www.zkb.ch/de/pr/pk/efinance/ebanking/login-verfahren.html
-
```
Comment (stephengroat) [inline _data/banking.yml:669]: we use newlines at the end of files, please remove change

File: _data/food.yml [hunk 3]
```diff
@@ -11,6 +11,13 @@ websites:
       tfa: No
       twitter: maxhamburgare
 
+    - name: Morrisons
```
Comment (stephengroat) [inline _data/food.yml:14]: Please describe how this site fits the criteria

**Review by stephengroat — CHANGES_REQUESTED** [review]
Please remove the natwest and `banking` changes and describe how the sites meets the criteria

File: _data/food.yml (hunk 3 above)
Comment (StevenChorkley) [inline _data/food.yml]: Why wouldn't it? It's an online supermarket?

Comment (djrarky) [reply _data/food.yml:14]: Most supermarket accounts including Morrisons holds personal information such as bank details and addresses. [I'd say that fits 1 and 3.](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria)

**Pull request closed without merging**